Fst2List.cpp

```diff
@@ -1979,18 +1983,39 @@ int CFstApp::outWordsOfGraph(int depth) {
   return 0;
 }
 
-//
 //
 //
 //
 
-// TODO(jhondoe) Use UnitexGetOpt instead argc, argv
+const char* optstring_Fst2List=":o:Sp:a:t:l:i:mdf:vVhs:r:c:";
+const struct option_TS lopts_Fst2List[]= {
```

Comment:
A suggestion: For long options, instead of `foo-bar` use `foo_bar`, e.g. rather  than `io-separator` use `io_separator`. This is more consistent with the convention adopted by other modules. FYI, UnitexGetOpt can then parse `foo-bar` as `foo_bar` (see [here](https://github.com/UnitexGramLab/unitex-core/blob/23cf3926fd6d1f532e07d74fa2f08fd7107054f2/UnitexGetOpt.cpp#L155))